presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTypeHandle.java

```diff
@@ -80,11 +80,17 @@ public int getJdbcType()
     }
 
     @JsonProperty
-    public int getColumnSize()
+    public Optional<Integer> getColumnSize()
     {
         return columnSize;
     }
 
+    @JsonIgnore
+    public int getRequiredColumnSize()
```

Comment:
Consider to revert condition in naming:
```
int getRequiredColumnSize() -> int getColumnSize() 
Optional<Integer> getColumnSize() -> Optional<Integer> getOptionalColumnSize()
```
As it is more inline with standard java naming. 
And according to commit message: `The COLUMN_SIZE field is null in **some** cases`, so only in that some cases we will have longer getOptionalColumnSize()

Reply:
consistent with https://github.com/prestosql/presto/blob/ece62ab29bc96a723669687e17bc2eef183a2d4d/presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTypeHandle.java#L88-L98

also, since the new `getRequiredColumnSize` is not a drop in replacement for the old method, i want anyone using this class to update their code, rather than blindly continue to call same method, but with a different behavior.

Reply:
preexisting, so let's keep compatible with existing name for  now and we can change both if we want to

other name proposal is `getExisting...`....  not sure.

Reply:
> And according to commit message: The COLUMN_SIZE field is null in **some** cases, so only in that some cases we will have longer getOptionalColumnSize()

That's interesting, but i am concerned about implicit validating happening within a method that _looks like_ ordinary getter ("getColumnSize").

Reply:
I had the same idea like @jirassimok -> but had the feeling it could mislead a user a little bit as well -> and maybe with such name it's already better to inline this method -> but there are a lot of occurrences -> goto begin.

But as long this idea came to someone else I'd vote for it.

Reply:
I personally do not like "OrThrow" in method names. I want the method to declare semantics ("something is required"), rather than describe implementation ("i will throw").
Anyway, i think both names are OK names, but one is existing (getRequiredDecimalDigits) and I do not want to change this within this PR.
The change `int getColumnSize` to `Optional getColumnSize` is already a problem to accommodate to for anyone depending on this code.